easybuild/tools/package/utilities.py

```diff
@@ -47,7 +47,7 @@
 from easybuild.tools.run import run_cmd
 from easybuild.tools.toolchain import DUMMY_TOOLCHAIN_NAME
 from easybuild.tools.utilities import import_available_modules, quote_str
-
+from subprocess import list2cmdline
```

Comment:
I actually removed quote_str as well since I no longer use it with the cmd list version.